test/integration/test-boilerplates.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {createFixtureIframe, expectBodyToBecomeVisible} from
+    '../../testing/iframe.js';
+import {getStyle} from '../../src/style';
+
+describe('Opacity Boilerplate', () => {
+
+  let fixture;
+  beforeEach(() => {
+    return createFixtureIframe('test/fixtures/boilerplate-opacity.html', 1000)
```

Comment:
Same with file names: let's make sure the version of the bolierplate is clear. OLD/NEW or V0/V1.